### DISCUSSION TITLE
The evaluation always out of time. Request run time log.

### DISCUSSION THREAD

**Discussion by mixiancmx** [PaddlePaddle/PaddleSpatial#272] [BaiduKDDCup2022] [answered]
![image](https://user-images.githubusercontent.com/83197963/179211456-27e5be73-4239-4e81-ba89-036eb729af10.png)

Hi, this is team CURE Lab Team.

We encounter the time-out error when evaluating on the platform. However, the evaluation can finish within **1 min** in our machine (a single TitanXP). We hope that you can provide an error log or run time log. We are so confused.

**Comment by VEWOXIC** [comment]
I am one of the team member. 
We only changed the num_workers from 1 to 0 comparing with the 16:40 submission which should works the same. However, the earlier one report out_of_memory in 10 min, but this one takes 2h to time out. I am not sure if this one is correctly run.

**Comment by ramber1836** [comment, marked as answer]
Pending timeout error is caused by the situation where limited resources were allocated but a large number of submissions are made at the same time.
If the Pending Timeout Error was encountered, no error log is available. 
You can resubmit your code and try it again. 
Sorry for that.
